File: app/src/main/java/com/mt/mtapi/LanguageUtil.java
package com.mt.mtapi;

import android.content.res.Configuration;
import android.util.DisplayMetrics;
import java.util.Locale;

/* loaded from: classes3.dex */
public class LanguageUtil {
    public static void set(boolean isEnglish) {
        Configuration configuration = mtDevice.ctx.getResources().getConfiguration();
        DisplayMetrics displayMetrics = mtDevice.ctx.getResources().getDisplayMetrics();
        if (isEnglish) {
        } else {
            configuration.locale = Locale.SIMPLIFIED_CHINESE;
        }
        mtDevice.ctx.getResources().updateConfiguration(configuration, displayMetrics);
    }
}

File: app/src/main/java/com/mt/mtapi/LanguageUtil.kt
package com.mt.mtapi

import android.content.res.Configuration
import android.util.DisplayMetrics
import java.util.Locale


object LanguageUtil {
    fun set(isEnglish: Boolean) {
        val configuration: Configuration = mtDevice.Companion.ctx!!.resources.configuration
        val displayMetrics: DisplayMetrics? =
            mtDevice.Companion.ctx!!.resources.displayMetrics
        if (isEnglish) {
            configuration.locale = Locale.ENGLISH
        } else {
            configuration.locale = Locale.SIMPLIFIED_CHINESE
        }
        mtDevice.Companion.ctx!!.resources.updateConfiguration(configuration, displayMetrics)
    }
}
